lib/streamlit/MediaFileManager.py

```diff
@@ -28,6 +28,11 @@
 
 STATIC_MEDIA_ENDPOINT = "/media"
 
+# Length of time (seconds) to keep media files so that we don't pull the rug out from
+# under rapid-media-generating elements, e.g. using a slider to produce pyplots.
+# See Issues #1440, #1445, #1294
+KEEP_DELAY = 2
```

Comment:
Rename to `KEEP_DELAY_SEC` so the unit is obvious when reading the variable name.